pkg/build/strategies/sti/sti.go

```diff
@@ -599,12 +607,20 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 				continue
 			}
 			// The log level > 3 forces to use glog instead of printing to stdout
-			if glog.V(3) {
+			if glog.Is(3) {
```

Comment:
Agree in practice, but for this particular case, the Is(3) also protects a continue statement which changes the behavior of the loop.